### PR TITLE
tutorials/setup: fix broken link

### PR DESCRIPTION
**Pull request by haircommander**
Signed-off-by: Peter Hunt <pehunt@redhat.com>

### CODE REVIEW

**Comment by saschagrunert** [comment]
LGTM

File: tutorials/setup.md [hunk 1]
```diff
@@ -288,6 +288,5 @@ sudo systemctl start crio
 ## Using CRI-O
 
 - Follow this [tutorial](crictl.md) to quickly get started running simple pods and containers.
-- To run quickly with kubernetes, see [our quick kuberentes guide](quick-kubernetes.md).
```
Comment (TomSweeneyRedHat) [inline tutorials/setup.md]: So where did this go to?  Is it really of no value?

**Comment by TomSweeneyRedHat** [comment]
Looks like this PR is awaiting an `/approve`

File: tutorials/setup.md (hunk 1 above)
Comment (haircommander) [reply tutorials/setup.md]: The contents were moved to the README so users could quickly see how to run it, which made the (already very small) tutorial redundent

**Pull request merged**